Assets/MixedRealityToolkit/_Core/Utilities/EdgeUtilities.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    public static class EdgeUtils
```

Comment:
EdgeUtility?

Reply:
Matches some of our other utils classes (ex: MathUtils), but I am happy to change

Reply:
I don't mind updating those other ones.

Reply:
sounds good. i'll change this one.